### PR TITLE
Correct links

### PR DESCRIPTION
**Pull request by callea2**
PT Story: https://www.pivotaltracker.com/story/show/122126941

Fixed the most important links

### CODE REVIEW

**Comment by tochman** [comment]
Looks good to me atm, I would like the Login and Sign up links to activate the modal but that is a separate story, I would also like to condition the display of those links to when a user is NOT logged in. After login these should be hidden and replaced with a Sign out link. At the very least. Should that be included in this PR?

**Comment by callea2** [comment]
Yeah, I can fix that

File: features/step_definitions/user_authentication_steps.rb [hunk 1]
```diff
@@ -3,3 +3,13 @@
...
 end
+
+Given(/^I am signed in$/) do
+  steps %Q{
+    Given I am on the "login page"
+    And I fill in "Email" with "calle@gmail.com"
+    And I fill in "Password" with "password"
+    And I click "Log in"
+    Then I should be on the "home page"
+  }
```
Comment (tochman) [inline features/step_definitions/user_authentication_steps.rb]: This way of logging in a user is perfectly fine. There is, however, another, faster way using the Devise helpers. I showcased that in the Rails messaging project. 

Not a blocker though. You can keep this.

**Comment by tochman** [comment]
Plz add a new story to PT for the modal login.

File: features/support/warden.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Warden.test_mode!
+World Warden::Test::Helpers
+After { Warden.test_reset! }
```
Comment (tochman) [inline features/support/warden.rb:3]: I'm impressed @callea2 Great job!!

**Comment by tochman** [comment]
Imo this is as good as it gets. Ready to merge.

**Pull request merged**